### PR TITLE
April serivces adjustment

### PR DESCRIPTION
**Pull request by bigglesandginger**
# Description

Please include a summary of the changes and the related issue.

Resolves #(issue)

### CODE REVIEW

File: kubernetes/deploy.sh
Comment (YohannParis) [inline kubernetes/deploy.sh:1]: Why the double `shift`?

File: kubernetes/overlays/prod/overlays/askem-staging/services/skema/skema-text-reading-deployment.yaml
Comment (YohannParis) [inline kubernetes/overlays/prod/overlays/askem-staging/services/skema/skema-text-reading-deployment.yaml:1]: Do we still need `skema-text-reading` to be on its own node?
This is just a matter of speed right? or does it needs that full node to be able to run?

**Review by YohannParis — CHANGES_REQUESTED** [review]
Let's wait post-evaluation

File: kubernetes/deploy.sh
Comment (bigglesandginger) [reply kubernetes/deploy.sh:1]: `shift` in bash moves the parameter pointer along (effectively renumbering the parameters ignoring the shifted ones).  This is useful when looping over the command arguments (basically moving to the next one).  However, you can see that sometimes (all the time) I get the Environment (which is the next parameter) and so must call an additional `shift` (there is a `shift` at the bottom of the loop, the done statement, as well for the current parameter) and in the case of encrypting or decrypting a single file I need to call yet another `shift`.  The code could have been written as 
```
COMMAND="encrypt"
ENVIRONMENT="$2"
shift
FILE_NAME="$2"
shift
;;
```
a question of which looks more ugly in an ugly language :)

File: kubernetes/overlays/prod/overlays/askem-staging/services/skema/skema-text-reading-deployment.yaml
Comment (bigglesandginger) [reply kubernetes/overlays/prod/overlays/askem-staging/services/skema/skema-text-reading-deployment.yaml:1]: `skema-text-reading` uses 8GB RAM on start up - does it use more? - the previous version did use approx 20GB RAM

Comment (bigglesandginger) [reply kubernetes/overlays/prod/overlays/askem-staging/services/skema/skema-text-reading-deployment.yaml:1]: and this is definitely a question of RAM not cpu

File: kubernetes/overlays/prod/overlays/askem-production/hmi/server/hmi-server-deployment.yaml [hunk 1]
```diff
@@ -6,14 +6,16 @@ metadata:
...
-      tolerations:
-        - key: "workload"
-          operator: "Equal"
-          value: "hmi-server"
-          effect: "NoSchedule"
+      # tolerations:
+      #   - key: "workload"
+      #     operator: "Equal"
+      #     value: "hmi-server"
+      #     effect: "NoSchedule"
```
Comment (YohannParis) [inline kubernetes/overlays/prod/overlays/askem-production/hmi/server/hmi-server-deployment.yaml:13]:
```suggestion
```

**Review by YohannParis — APPROVED** [review]

File: kubernetes/overlays/prod/overlays/askem-production/hmi/server/hmi-server-deployment.yaml (hunk 1 above)
Comment (bigglesandginger) [reply kubernetes/overlays/prod/overlays/askem-production/hmi/server/hmi-server-deployment.yaml:13]: I have no idea what is being suggested here - when I commented out the lines VIM added the hashes - but honestly I think we should delete these...

**Pull request merged**